h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -606,6 +606,7 @@ public static Vec makeVec(long [] vals, String [] domain, Key<Vec> vecKey){
    *  and initialized to the given constant value.  */
   public Vec makeCon(final double d) { return makeCon(d, group(), _rowLayout, T_NUM); }
   public Vec makeCon(final double d, byte type) { return makeCon(d, group(), _rowLayout, type); }
+  public Vec makeCon(final long l, byte type) { return makeCon(l, null, group(), _rowLayout, type); }
```

Comment:
what is the difference?

Reply:
there was no way how to create a String vec with the same layout as the current vec